tests/interpolation_test.py

```diff
@@ -15,7 +15,6 @@
 neutrino_scattering = CrossGeneratingInteraction(
     particles=((neutrino, neutrino), (neutrino, neutrino)),
     antiparticles=((False, True), (False, True)),
-    decoupling_temperature=0 * UNITS.MeV,
```

Comment:
What's up with this?

Reply:
I just remember now. I think you changed it to wash_out_temperature. It is set to 0 by default, right?

Reply:
Yes, I don't think we ever use it